plugins/modules/packaging/language/cargo.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 Radek Sprta <mail@radeksprta.eu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: cargo
+short_description: Manage Rust packages with cargo
+description:
+  - Manage Rust packages with cargo.
+author: "Radek Sprta (@radek-sprta)"
+options:
+  name:
+    description:
+      - The name of a Rust package to install.
+    type: str
```

Comment:
Does it make sense to allow installing a list of packages? Most package managing modules allow this. This would then have to change to
```suggestion
    type: list
    elements: str
```

Reply:
Yea, cargo can install multiple packages, so I'll refactor the code to accept list.